1.1/MetadataGranularities/README.md

```diff
@@ -0,0 +1,200 @@
+# Metadata Granularities
+
+This sample demonstrates the use of [Metadata in 3D Tiles 1.1](https://github.com/CesiumGS/3d-tiles/blob/b7fa517e88149942bc6166b11944bac67565369e/specification/README.md#metadata) on different levels of granularity. The sample consists of a tileset with 4 tiles where each tile has 5 contents, and the contents are assigned to two different groups. Metadata is assigned to the tileset, the tiles, each content, and to the groups.
+
+## Screenshot
+
+![MetadataGranularities](screenshot/MetadataGranularities.png)
+
+## Sandcastle Code
+
+```JavaScript
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+// Create the tileset, and set its model matrix to move it
+// to a certain position on the globe
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'http://localhost:8003/1.1/MetadataGranularities/tileset.json',
+    maximumScreenSpaceError: 1,
+    debugShowBoundingVolume: true
+}));
+tileset.modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(
+  Cesium.Cartesian3.fromDegrees(-75.152325, 39.94704, 0.0));
+
+// Create the label that will display metadata information
+var labelEntity = viewer.entities.add({
+  label: {
+    showBackground: true,
+    font: "14px monospace",
+    horizontalOrigin: Cesium.HorizontalOrigin.LEFT,
+    verticalOrigin: Cesium.VerticalOrigin.TOP,
+    pixelOffset: new Cesium.Cartesian2(15, 0),
+    // Workaround to always render label on top:
+    disableDepthTestDistance: Number.POSITIVE_INFINITY
+  }
+});
+
+// Adds metadata information to the label, with 
+// the given title indicating which granularity
+// of metadata this is. The given metadata object
+// can be a TilesetMetadata, TileMetadata, 
+// GroupMetadata, or ContentMetadata.
+var addMetadataToLabel = function(title, metadata) {
+  if (!Cesium.defined(metadata)) {
+    labelEntity.label.text += "(No " + title + ")\n";
+    return;
+  }
+  var propertyKeys = metadata.getPropertyIds();
+  if (!Cesium.defined(propertyKeys)) {
+    labelEntity.label.text += "(No properties for " + title + ")\n";
+  }
+  labelEntity.label.text += title + ":\n";
+  for (var i=0; i<propertyKeys.length; i++) {
+    var propertyKey = propertyKeys[i];
+    var propertyValue = metadata.getProperty(propertyKey);          
+    //console.log("propertyKey", propertyKey);
+    //console.log("propertyValue", propertyValue);
+    labelEntity.label.text += "  " + propertyKey + ": "+ propertyValue;
+    labelEntity.label.text += "\n";
```

Comment:
[template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) will look nicer for a lot of these string builders